scripts/config/lib/ngx_mruby/routes_path.rb

```diff
@@ -0,0 +1,33 @@
+# ghetto require, since mruby doesn't have require
+eval(File.read('/app/bin/config/lib/nginx_config_util.rb'))
+
+USER_CONFIG = "/app/static.json"
+
+def match_excepts(excepts, uri)
+  return false unless excepts
+
+  excepts.each do |except|
+    return true if Regexp.compile("^#{except}$") =~ uri
```

Comment:
Not every call does this. The [proxy matching stuff does not](https://github.com/hone/heroku-buildpack-static/blob/57125c2464476bba14525bed90a72542750b5e24/scripts/config/lib/nginx_config_util.rb#L42).